src/traces_layouts.jl

```diff
@@ -2,7 +2,7 @@ abstract AbstractTrace
 abstract AbstractLayout
 
 type GenericTrace{T<:Associative{Symbol,Any}} <: AbstractTrace
-    kind::ASCIIString
+    kind::String
```

Comment:
This is a breaking change but it looks reasonably private so it should be fine it it passes CI.